packages/notification-service/package.json

```diff
@@ -19,6 +19,8 @@
     "deploy": "./deploy.sh"
   },
   "dependencies": {
+    "@celo/utils": "^0.0.6-beta5",
```

Comment:
Any reason we need a beta5 version? Should we release a 0.0.7 🔫

Reply:
This is the most recent version, copied over from `/mobile` to be consistent
Looks like we've been making small changes to version 0.0.6 and releasing it as different betas:
https://www.npmjs.com/package/@celo/utils